runtime.md

```diff
@@ -1,5 +1,47 @@
 # Runtime and Lifecycle
 
+## State
+
+The runtime state for a container is persisted on disk so that external tools can consume and act on this information.
+The runtime state is stored in a JSON encoded file.
+It is recommended that this file is stored in a temporary filesystem so that it can be removed on a system reboot.
+On Linux based systems the state information should be stored in `/run/oci/containers`.
+The directory structure for a container is `/run/oci/containers/<containerID>/state.json`.
```

Comment:
After the meeting, my understanding was that we wanted a  flat `/run/oci/{containerID}/state.json` namespace, with internal-to-the-state content for figuring out the semantics of the state content.  If the `state.json` requires a version field that clarifies the associated spec (e.g. [my suggestion here][1]), there would be no need for the `containers/` path to namespace different types of container specs.

[1]: https://github.com/opencontainers/specs/issues/110#issuecomment-133587646